user/views.py

```diff
@@ -0,0 +1,124 @@
+# autopep8: off
+import os
+import json
+import bcrypt
+import re
+import jwt
+import my_settings
+import utils
+
+from datetime               import datetime
+from django.views           import View
+from user.models            import User
+from django.http            import JsonResponse
+from django.db              import IntegrityError
+
+class SignUp(View):
+    def post(self, request):
+        try:
+            data  = json.loads(request.body)
+            regex = '^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$'
+            if not (re.search(regex, data.get('email'))):
+                return JsonResponse({"message": "INVALID_EMAIL"}, status=400)
+
+            if User.objects.filter(account=data.get('account')).exists():
+                return JsonResponse({"message": "USER_ID_TAKEN"}, status=400)
+
+            password       = data.get('password').encode('utf-8')
+            password_crypt = bcrypt.hashpw(password, bcrypt.gensalt())
+            password_crypt = password_crypt.decode('utf-8')
```

Comment:
```suggestion
            password_crypt = bcrypt.hashpw(password, bcrypt.gensalt()).decode('utf-8')
```
굳이 두 번에 분리하시지 않고 한 번에 처리하셔도 좋습니다

Reply:
싱글라인으로 수정했습니다 :)